src/sinks/datadog/traces/config.rs

```diff
@@ -98,7 +98,6 @@ pub enum DatadogTracesEndpoint {
 /// Store traces & APM stats endpoints actual URIs.
 pub struct DatadogTracesEndpointConfiguration {
     traces_endpoint: Uri,
-    // Unused so far
     stats_endpoint: Uri,
```

Comment:
I can't find any usage of `stats_endpoint` in this PR. Has this deletion been intentional?

Reply:
Yes, here `stats_endpoint` holds the APM stats URL derived from the configuration, then an [APM stats request is created](https://github.com/vectordotdev/vector/blob/895a6a910149dda67db5b0b97bb5f630c774dc43/src/sinks/datadog/traces/request_builder.rs#L422) with the `DatadogTracesEndpoint::APMStats` as the targeted endpoint  that is ultimately use to [get the real URL](https://github.com/vectordotdev/vector/blob/895a6a910149dda67db5b0b97bb5f630c774dc43/src/sinks/datadog/traces/config.rs#L108).